readthedocs/core/symlink.py

```diff
@@ -63,7 +63,7 @@
 
 from readthedocs.builds.models import Version
 from readthedocs.core.utils.extend import SettingsOverrideObject
-from readthedocs.core.utils import safe_makedirs
+from readthedocs.core.utils import safe_makedirs, unlink
```

Comment:
Should this be called `safe_unlink` to mirror the `safe_makedirs` and show it has a bit more error handling.